packages/frontend/src/styles/monaco.css

```diff
@@ -2,3 +2,11 @@
     background-color: yellow;
     border-bottom: 2px solid red;
 }
+
+.tooltip-editor {
+    border: 1px solid #ccc;
+    overflow: visible;
+}
+.tooltip-editor .suggest-widget {
```

Comment:
Fix for suggestions

Before

![Screenshot from 2025-04-07 10-28-02](https://github.com/user-attachments/assets/77becb16-64d8-4354-8707-d3fb563ea797)

After

![image](https://github.com/user-attachments/assets/ffcdc4b8-ef22-4a3f-922d-5a0ccc712c78)